src/files-and-videos/videos-page/data/utils.js

```diff
@@ -49,10 +48,11 @@ export const updateFileValues = (files) => {
       id: edxVideoId,
       wrapperType,
       dateAdded: created.toString(),
+      usageLocations: isNewFile ? [] : RequestStatus.LOADING,
```

Comment:
RequestStatus has no `LOADING` attribute, so this evaluates to `undefined`. Anyway isn't it an inconsistent data type to say "`usageLocations` is either an array or a string that says what the requestStatus is"?

You could change this to an object with `value` and `status`, or maybe just explicitly set `null` instead of `RequestStatus.LOADING`.

Reply:
Thank you for catching this! I meant to use `RequestStatus.IN_PROGRESS`. I will update it to be `null` and show then spinner when the `usageLocations` is `null`.